src/ast/scopes/Scope.ts

```diff
@@ -86,7 +86,7 @@ export default class Scope {
 			const declaration = this.variables[key];
 
 			// we can disregard exports.foo etc
-			if (declaration.exportName && declaration.isReassigned && !declaration.isId)
+			if (declaration.exportName && (declaration.isDefault || declaration.isReassigned && !declaration.isId))
```

Comment:
The non-use of more parentheses here makes me a little uneasy (sorry, but I'm one of those people who need to look into an operator precedence table to see that `&&` binds more strongly than `||`).